internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -18,9 +18,13 @@ var (
 	atlasRegion  = "US_EAST_1"
 )
 
+func TestMain(m *testing.M) {
+	acc.TestMainExecution(m)
```

Comment:
don't handle the exist status internally let `TestMain` handle that which is supposed to do

Reply:
it's a bit more verbose but I like the idea and it's more explicit, will change to something like:
```
func TestMain(m *testing.M) {
	acc.SetupSharedResources()
	exitCode := m.Run()
	acc.CleanupSharedResources()
	os.Exit(exitCode)
}
```

thx!